doc/metric_spec.md

```diff
@@ -0,0 +1,111 @@
+2. [drbd](#drbd)
+3. [sbd](#sbd)
+4. [corosyncl](#corosync)
+
+# Pacemaker 
+
+The Pacemaker cluster metrics are atomic metrics and represent and updated snapshot of the HA cluster, retrieved fetching the XML CIB of pacemaker.
+
+Some of the pacemaker metrics like `ha_cluster_node_resources` and `ha_cluster_nodes` metrics with labels share a common trait:
+ 
+they can be either set to `1` or they are absent, this is because they track the real state of the cluster resources monitored.
+
+1. [ha_cluster_node_resources](#ha_cluster_node_resources)
+2. [ha_cluster_nodes](#ha_cluster_nodes)
+3. [ha_cluster_nodes_configured_total](#ha_cluster_nodes_configured_total)
+4. [ha_cluster_resources_configured_total](#ha_cluster_resources_configured_total)
+
+
+
+## ha_cluster_node_resources
+
+This metric show the current status of  a node resource. 
+
+A resource that previously was in the cluster but isn't anymore, will not monitored. Example:
+
+```ha_cluster_node_resources{managed="true",node_name="1b115",resource_name="cluster_md",role="started",status="active"} 1```
+
+The metric will absent and not `0`
+
+
+All the values are 1:1 with Pacemaker schema.
+
+- `managed`: indicates `true` or `false` if the resource is managed in cluster
+- `node_name`: name of node of cluster monitored
+- `resource_name`: name of pacemaker resource
+- `role`:  allowed values `Started/Stopped/Master/Slave` which are same as pacemaker roles for resources.
+- `status` allowed values `active/orphaned/blocked/failed/failureIgnored/` status of resource from pacemaker XML
+
+Example:
+
+```
+ha_cluster_node_resources{managed="true",node_name="1b115",resource_name="cluster_md",role="started",status="active"} 1
+ha_cluster_node_resources{managed="true",node_name="1b115",resource_name="clvm",role="started",status="active"} 1
+ha_cluster_node_resources{managed="true",node_name="1b115",resource_name="dlm",role="started",status="active"} 1
+ha_cluster_node_resources{managed="true",node_name="1b115",resource_name="drbd_passive",role="master",status="active"} 1
+ha_cluster_node_resources{managed="true",node_name="1b115",resource_name="fs_cluster_md",role="started",status="active"} 1
+ha_cluster_node_resources{managed="true",node_name="1b115",resource_name="fs_drbd_passive",role="started",status="active"} 1
+ha_cluster_node_resources{managed="true",node_name="1b115",resource_name="stonith-sbd",role="started",status="active"} 1
+ha_cluster_node_resources{managed="true",node_name="1b115",resource_name="vg_cluster_md",role="started",status="active"} 1
+ha_cluster_node_resources{managed="true",node_name="1b211",resource_name="dlm",role="started",status="active"} 1
+ha_cluster_node_resources{managed="true",node_name="1b211",resource_name="fs_cluster_md",role="stopped",status="active"} 1
+ha_cluster_node_resources{managed="true",node_name="1b211",resource_name="vg_cluster_md",role="stopped",status="active"} 1
+```
+
+## ha_cluster_nodes
+
+- `node_name`: name of node of cluster
```

Comment:
Would "name of cluster node" probably be better?

Reply:
yop